dev/local-environment/setup.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-PARTNER_CHAINS_NODE_IMAGE="ghcr.io/input-output-hk/partner-chains/partner-chains-node:v1.5.0"
+PARTNER_CHAINS_NODE_IMAGE="ghcr.io/input-output-hk/partner-chains/partner-chains-node-unstable:latest"
```

Comment:
Local-env tests are run in pre-merge, but unstable image is pushed only in `build-linux-x86_64-post-merge`